data/slack-jackson-dto-test-extensions/src/main/kotlin/io/olaph/slack/dto/jackson/group/im/MessageExtension.kt

```diff
@@ -0,0 +1,3 @@
+package io.olaph.slack.dto.jackson.group.im
+
+fun SuccessfulImHistoryResponse.Message.Companion.sample() = SuccessfulImHistoryResponse.Message("", "")
```

Comment:
no need for an own file since its a nested object, please put it in the ImHistoryExtension.kt file